modules/heat_conduction/src/constraints/ModularGapConductanceConstraint.C

```diff
@@ -135,17 +137,15 @@ ModularGapConductanceConstraint::initialSetup()
 
   // Select proper coordinate system and geometry (plate, cylinder, sphere)
   setGapGeometryParameters(
-      _pars, coord_system, feProblem().getAxisymmetricRadialCoord(), _gap_geometry_type, _p1, _p2);
+      _pars, coord_system, feProblem().getAxisymmetricRadialCoord(), _gap_geometry_type);
 }
 
 void
 ModularGapConductanceConstraint::setGapGeometryParameters(
     const InputParameters & params,
     const Moose::CoordinateSystemType coord_sys,
     unsigned int axisymmetric_radial_coord,
-    GapGeometry & gap_geometry_type,
-    Point & p1,
-    Point & p2)
+    GapGeometry & gap_geometry_type)
```

Comment:
We have similar functions in other formulations for gap heat transfer in MOOSE and apps, e.g. in the older gap heat transfer note-to-segment and specialized physics: Should these code changes be extended to those cases for consistency?

Reply:
I suggest doing that in a separate PR. There might be fallout from this API change in the other cases.